Perform-String-Shifts.py

```diff
@@ -51,3 +52,14 @@ def stringShift(self, s: str, shift: List[List[int]]) -> str:
             return s[-direction:] + s[:-direction]
         else:
             return s
+
+# Method 2: Input in the form of list
+class Solution:
+    def stringShift(self, s: str, shift: List[List[int]]) -> str:
+        s_list = [i for i in s] #Changing String to List and carrying out
+        for i in range(len(shift)):
+            if shift[i][0]:
+                s_list = s_list[-(shift[i])[1]:] + s_list[:-(shift[i][1])]
+            else:
+                s_list = s_list[(shift[i][1]):] + s_list[:(shift[i][1])] 
```

Comment:
You're shifting for every shift which is not optimized. Also, add a comment for time and space complexity.